_posts/2018-12-03-how-to-make-golden-egg-fried-rice.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "How To Make Golden Fried Rice"
+date: 2018-12-03 21:20:12
+tags:
+- Food
+- chinese cuisine
+categories: food recipes
+twitter_text: "How To Make Golden Fried Rice"
+authors: Cheng Yang
+image: https://c1.staticflickr.com/3/2466/3708994257_4e359233c9_b.jpg
+image_url: https://www.flickr.com/photos/35856236@N06/3708994257/
+image_credit: hawaiianfoodrecipe
+---
+
+Egg Fried Rice is a very homely dish in China. Every Chinese family make their fried rice differently, so you don't actually need a receipt to make fried rice, you can add whatever you like into the rice. Today, I am going to show you my method to make this simple, quick and delicious dish.
+
+## Ingredients
+- 400g cooked rice (For two) - I use `Nishiki Medium Grain Rice` (https://www.amazon.com/Nishiki-Medium-Grain-Rice-Pound/dp/B00852ZN2U/ref=sr_1_8_s_it?s=grocery&ie=UTF8&qid=1543877113&sr=1-8&keywords=rice)
+- 3 eggs
+- half onion (Chopped)
+- 80 gram luncheon meat / any other meats
+- corns and green peas
+- 1/2 tbp salt
+- 1/2 tbp sugar
+- 3/4 tbp white pepper powder
+- 4 Thai Pepper (Optional)
+- appropriate amount of scallion
+
+
+## Tips
+- In order to make the best egg fried rice, I suggest to use leftover rice or you can put the cooked rice into refrigerated for at least 2 hours.
+## DIRECTIONS:
+- Chop all the ingredients into small pieces.
+- In a working pan, Heat oil on high heat. Saute the onion and Thai Pepper (Only if you like spicy) until they start to soften, add the luncheon meat, corns and green peas, saute them for 3 mins. Once done, Put them in a separate dish. We will use them later.
+- Put cooked rice into a Mixing Bowl, add eggs one by one. Mix the rice with the egg yolks, make sure all the rice grain are separate.
+- In a working pan, Heat oil on high heat. Add the rice, saute it until the egg is cooked, add all the ingredients, season with salt, sugar, and white pepper. Mix well with all the ingredients, and you can add more salt and pepper to adjust seasoning.
+- Finally, You can put some green scallion on the top of the rice. Then The Golden Fried Rice is ready.
```

Comment:
This might be better as `Finally, garnish with sliced green scallion.` And I think `Then Golden Fried Rice is ready` needs a little more character. I love the final line `First we eat, then we do everything else`! I think this final line of the post should have a little more character.